sql/postgresql/README.md

```diff
@@ -25,7 +25,73 @@ createdb my_pap_db
 psql --dbname=my_pap_db --file="<pingauthorize-contrib-root>/sql/postgresql/init-policy-database.sql" >/dev/null
 ```
 You should now have an initialized policy database. For instructions on configuring the Policy Editor to connect to
-your PostgreSQL instance, see "[Installing the PingAuthorize Policy Editor manually](https://docs.pingidentity.com/csh?Product=paz-latest&context=paz_install_pe_noninteractive)".
+your PostgreSQL instance, see [Installing the PingAuthorize Policy Editor manually](https://docs.pingidentity.com/csh?Product=paz-latest&context=paz_install_pe_noninteractive).
+
+
+### Upgrading an existing policy database to a newer schema
+1. Verify the current version of your policy database using one of the following methods.
+
+**Optional** - Access `my_pap_db` and query the `TAG` column of `my_pap_db.databasechangelog`
+
+```
+psql -U postgres -d my_pap_db
+```
+
+```
+SELECT TAG FROM DATABASECHANGELOG WHERE TAG IS NOT NULL ORDER BY TAG DESC LIMIT 1;
+```
+
+**Optional** - Follow the [steps to non-interactively set up](https://docs.pingidentity.com/csh?Product=paz-latest&context=paz_install_pe_noninteractive)
+   the target version of the Policy Editor. If your schema is out of date, you will be prompted to run the intermediate upgrade scripts sequentially.
+
+```
+bin/setup demo \
+  --adminUsername admin \
+  --dbConnectionString "jdbc:postgresql://<host>:<postgresql_port>/my_pap_db" \
```

Comment:
This is a nit, but the PostgreSQL host does not necessarily need to be the same as the server host.